src/Microsoft.DotNet.Darc/Darc/Operations/GetAssetOperation.cs

```diff
@@ -49,7 +55,12 @@ public override async Task<int> ExecuteAsync()
             List<Asset> matchingAssets =
                 (await remote.GetAssetsAsync(name: _options.Name, version: _options.Version, buildId: _options.Build)).ToList();
 
-            var queryDescription = new StringBuilder($"name '{_options.Name}'");
+            var queryDescription = new StringBuilder();
+
+            if (!string.IsNullOrEmpty(_options.Name))
+            {
+                queryDescription.Append($" named '{_options.Name}'");
```

Comment:
If the command is run without this option, and with the build, the message will be a bit weird, for example `Looking up assets and version 1.0.0...`

Reply:
Maybe just rearranging the `ifs` a bit, swapping the positions of the build and versions one

Reply:
Fixed